### PR TITLE
in-app design review

### PR DESCRIPTION
**Pull request by jyothishnt**
## Type

- [ ] Bug fix
- [ ] New feature
- [x] Improvement to existing feature

## Related JIRA Issue(s)
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1251

## Description
Design review of in-app-gene search

## Deployment URL
http://in-app-search-design-review.review.ensembl.org/

## Views affected
In-app gene search

### Other effects
NA

- [ ] I have checked any requirements for Google Analytics
- [ ] I have created any required tests
- [ ] The PR has as its final commit a WASM/JS update commit if any Rust files were updated.

## Possible complications
NA

### CODE REVIEW

**Comment by ens-ma7** [comment]
The review URL isn't working.

File: src/ensembl/src/shared/components/question-button/QuestionButton.scss [hunk 1]
```diff
@@ -17,6 +17,8 @@
   width: 16px;
   height: 16px;
   padding: 0;
+  height: 14px;
```
Comment (ens-ma7) [inline src/ensembl/src/shared/components/question-button/QuestionButton.scss:20]: Looks like you didnt remove/overwrite, the width and height above. width and height are specified twice :)

**Review by ens-ma7 — COMMENTED** [review]
Ignore this. I press the wrong button

File: src/ensembl/src/shared/components/question-button/QuestionButton.scss (hunk 1 above)
Comment (ens-ma7) [reply src/ensembl/src/shared/components/question-button/QuestionButton.scss:18]: sorry @jyothishnt just realised that the default height and width is 14px so we dont need to specify the width and height here.  do you agree?

Comment (ens-ma7) [reply src/ensembl/src/shared/components/question-button/QuestionButton.scss:18]: sorry ignore the above comment, i was looking at the wrong question icon but i notice that you can specify just the height or the width to 14px and it works, you dont have to specify both but probably best to specify both to be safe.

**Review by ens-ma7 — APPROVED** [review]

**Comment by ens-ma7** [comment]
@jyothishnt i have approved the code, it looks fine to me. I leave setting the width and height for you to decide. either way is working fine.

**Review by azangru — APPROVED** [review]

**Pull request merged**